### ISSUE TITLE
trend看train_loss，训练过程中无曲綫，横坐标为0到1

### ISSUE THREAD

**Issue by SangerY** [fastnlp/fitlog#38]
想试用下fitlog工具加在tf-yolov3代码里，由于代码中未计算mAP，只想看下train_loss曲綫。加的代码如下:
fitlog.add_loss(train_step_loss, name="train_loss", step=train_step)
loss.log中信息如下：
![image](https://user-images.githubusercontent.com/12136347/98206572-1510f680-1f75-11eb-9e72-256db6e2db85.png)

在网页中打开是这样的
![image](https://user-images.githubusercontent.com/12136347/98206670-41c50e00-1f75-11eb-9229-51445a1986ce.png)

![image](https://user-images.githubusercontent.com/12136347/98206634-307c0180-1f75-11eb-9098-53ab5da62c62.png)

是我操作有误吗？

**Comment by yhcc** [comment]
感觉是由于那个数字是字符串的原因，建议你在fitlog.add_loss(float(train_step_loss), name="train_loss", step=train_step)添加一个float强制转换为数字试一下，add_loss只能识别torch.tensor, numpy, float类型。我之后研究下tf是啥数据类型。

**Comment by SangerY** [comment]
> 感觉是由于那个数字是字符串的原因，建议你在fitlog.add_loss(float(train_step_loss), name="train_loss", step=train_step)添加一个float强制转换为数字试一下，add_loss只能识别torch.tensor, numpy, float类型。我之后研究下tf是啥数据类型。

加了float就好了，谢谢。
下面这个图是正常的吗？这里是每个epoch的loss都叠加到steps上了吗，这样感觉不好看整体趋势，这样设计是有其他用处吗？
![image](https://user-images.githubusercontent.com/12136347/98339940-81f4c100-2047-11eb-9460-25e9701d7529.png)

**Comment by yhcc** [comment]
不，这样应该是出问题了。感觉应该是由于train_step重叠了？train_step应该是一个不断增长的值。

**Comment by SangerY** [comment]
> 不，这样应该是出问题了。感觉应该是由于train_step重叠了？train_step应该是一个不断增长的值。

嗯嗯，是的，我的train_step有问题。谢谢哈。
